### PR TITLE
feat: thesis topics from Kevin

### PR DESCRIPTION
**Pull request by kvn-myr**

### CODE REVIEW

**Comment by kvn-myr** [comment]
@M4GNV5

File: content/thesis-topics.md [hunk 1]
```diff
@@ -29,3 +29,12 @@ You will usually be supervised by the PhD students and graded + supervised by Pr
 - Analysis of the attack surface of infotainment systems via the broadcast interface [[2]](https://arstechnica.com/cars/2022/02/radio-station-snafu-in-seattle-bricks-some-mazda-infotainment-systems/)
 - Analysis of log files from automotive control units for forensic use
 - Fuzzing of Bluetooth implementations (A2DP, PBAP, SDAP, ...) 
+
+### Digital Forensics Topics 
+
+| Title | Short description | Contact person |
+| ----- | ----------------- | -------------- |
```
Comment (M4GNV5) [inline content/thesis-topics.md:36]: why is this new section a table, while the others are lists?
I would prefer either this one to be a list as well or the other ones to be converted to a table.

File: content/thesis-topics.md [hunk 2]
```diff
@@ -29,3 +29,12 @@ You will usually be supervised by the PhD students and graded + supervised by Pr
 - Analysis of the attack surface of infotainment systems via the broadcast interface [[2]](https://arstechnica.com/cars/2022/02/radio-station-snafu-in-seattle-bricks-some-mazda-infotainment-systems/)
 - Analysis of log files from automotive control units for forensic use
 - Fuzzing of Bluetooth implementations (A2DP, PBAP, SDAP, ...) 
+
+### Digital Forensics Topics 
+
+| Title | Short description | Contact person |
+| ----- | ----------------- | -------------- | 
+| File carving using Large Language Models (LLMs) | File carving of different memory dumps and using LLMs to identify traces | [kevin.mayer@carissma.eu](kevin.mayer@carissma.eu) |
```
Comment (M4GNV5) [inline content/thesis-topics.md:37]: Are you sure these links work? Shouldn't they be `mailto:` links?

**Review by M4GNV5 — CHANGES_REQUESTED** [review]

File: content/thesis-topics.md (hunk 2 above)
Comment (M4GNV5) [reply content/thesis-topics.md:37]: Consider using the included `cloakemail` plugin: `{{<cloakemail display="display text" address="email address">}}`

**Comment by kvn-myr** [comment]
As I indicated in Teams, we should provide additional information. So I would vote for the table-schema.

File: content/thesis-topics.md (hunk 1 above)
Comment (kvn-myr) [reply content/thesis-topics.md:36]: As I indicated in Teams, we should provide additional information. So I would vote for the table-schema.

File: content/thesis-topics.md (hunk 2 above)
Comment (kvn-myr) [reply content/thesis-topics.md]: done

**Review by M4GNV5 — APPROVED** [review]

**Pull request closed without merging**